avm/res/api-management/service/main.bicep

```diff
@@ -146,18 +146,33 @@ param policies array?
 @description('Optional. Portal settings.')
 param portalsettings array?
 
+// import { privateEndpointSingleServiceType } from 'br/public:avm/utl/types/avm-common-types:0.5.1'
+// @sys.description('Optional. Configuration details for private endpoints. For security reasons, it is recommended to use private endpoints whenever possible. Note: Private endpoints are supported with Developer, Basic, Standard, Premium, BasicV2, and StandardV2 SKUs only. Consumption SKU does not support private endpoints. \'virtualNetworkType\' must be set to \'None\' (or left empty which defaults to \'None\') on initial deployment of Private Endpoints.')
+// param privateEndpoints privateEndpointSingleServiceType[]?
```

Comment:
I guess this is supposed to be here for the next version when we figured out the private networking?